Core/WebViewController.swift

```diff
@@ -352,6 +357,7 @@ extension WebViewController: WKNavigationDelegate {
         let error = error as NSError
         if let url = webView.url, isHttpsUpgradeSite(url: url) {
             reportHttpsUpgradeSiteError(url: url, error: "\(error.domain)_\(error.code)")
+            statisticsStore.httpsUpgradesFailures += 1
```

Comment:
This needs to happen in both places (or just inside `reportHttpsUpgradeSiteError`).

Reply:
Good find! Thanks